users.js

```diff
@@ -7,9 +12,24 @@ async function addUser(email, password) {
 
     if (isUserExist.length === 0) {
       const hash = bcrypt.hashSync(password, 10);
+
+      const size = 200;
+      const value = email;
+
+      const png = jdenticon.toPng(value, size);
+
+      const avatar = path.join(

```

Comment:
move path to config

example: 

export const yourFunction = () => `text-${id}`

Reply:
done